src/applications/disability-benefits/all-claims/pages/medicalCare.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+function noCareCheck(errors, state, formData) {
+  const medicalCare = formData['view:medicalCare'];
```

Comment:
Can we use the `_.get()` method here to safely traverse these objects. (see other validation functions in `all-claims/validations.js`)

example:   `_.get('view:hasMilitaryRetiredPay', data, false);`